test/compflags/vass/callstack-internal-modules.chpl

```diff
@@ -0,0 +1,21 @@
+// https://github.com/chapel-lang/chapel/issues/17406
+
+// The goal of this code is to trigger a compilation error
+// in an internal module. Currently it uses + on (int,uint)
+// and relies on operator + to invoke _throwOpError() in this case.
+// If the implementation changes, another way to get a compilation error
+// in an internal module may be needed.
```

Comment:
I think you can make this more future-proof by using the `--prepend-internal-module-dir` compiler flag to either:
- replace an already-used module with a dummy module that will trigger this error, or
- add a new dummy module with a function that will trigger a compilation error and adjust ChapelStandard to also publicly use that dummy module.

But you definitely don't have to, it seems unlikely the code you're relying on will change.